web/packages/teleterm/src/ui/ConnectMyComputer/DocumentConnectMyComputerSetup/DocumentConnectMyComputerSetup.tsx

```diff
@@ -146,37 +147,40 @@ function AgentSetup({ rootClusterUri }: { rootClusterUri: RootClusterUri }) {
     downloadAgentAttempt,
     setDownloadAgentAttempt,
     agentProcessState,
+    logsFromStartTimeout,
   } = useConnectMyComputerContext();
   const cluster = ctx.clustersService.findCluster(rootClusterUri);
   const nodeToken = useRef<string>();
 
   const [createRoleAttempt, runCreateRoleAttempt, setCreateRoleAttempt] =
     useAsync(
-      useCallback(async () => {
```

Comment:
This version did not await `retryWithRelogin`, so throwing an error here was equal to unhandled error in a promise. Surprisingly though, the attempt status was set properly.